frontend/src/pages/api/gemini/chat.ts

```
@@ -0,0 +1,38 @@
+	if (!message || typeof message !== 'string' || message.trim().length === 0) {
+		return res.status(400).json({ error: 'Valid message is required' })
+	}
+
+	if (history && !Array.isArray(history)) {
+		return res.status(400).json({ error: 'History must be an array' })
+	}
+
+	try {
+		switch (req.method) {
+			case 'POST':
+				const chat = createChatSession(history)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)